spec/fpm/package/python_spec.rb

```diff
@@ -12,6 +12,18 @@ def python_usable?
     "'python' and/or 'easy_install' isn't in your PATH")
 end
 
+# Determine default value of a given easy_install's option
+def easy_install_default(option)
+  result = nil
+  execmd({:PYTHONPATH=>"#{example_dir}"}, 'python', :stderr=>false) do |stdin,stdout|
```

Comment:
Using "example_dir" here may look little-bit out of context, but it should be fine as long as the function is used within the ``describe FPM:Package::Python...`` block.